test/spec/properties.js

```diff
@@ -455,4 +455,27 @@ describe('properties', function() {
 
   });
 
+
+  describe('integration', function() {
+
+    var proxyConfig = {
+      get(target, property) {
+        // eslint-disable-next-line no-undef
```

Comment:
Proxy is marked as undefined by eslint because we use the babel ES6 plugin internally. They can't transpile proxies (https://babeljs.io/docs/en/learn#proxies), but we use the native one in testing.

Reply:
We could make Proxy as defined [here](https://github.com/bpmn-io/moddle/blob/master/test/.eslintrc).

Reply:
Or specify them as `global` elements: https://eslint.org/docs/user-guide/configuring/language-options#using-configuration-comments-1

Reply:
I specified them as `global` in the file. This is way cleaner, thanks for the hint :slightly_smiling_face: